api.php

```diff
@@ -409,6 +409,7 @@ public function run($userID, $permissionIsExplicit) {
 		$game = $this->getAssociatedGame();
 		$json = [
 			'anon' => $game->anon,
+			'date' => $game->datetxt($game->turn),
 			'drawType' => $game->drawType,
 			'excusedMissedTurns' => $game->excusedMissedTurns,
 			'gameOver' => $game->gameOver,
```

Comment:
I think it's better to process the year and season in PHP instead of splitting strings in JS.

Reply:
Lol explode is such an aggressive method name. Thanks for the writing this out!